content/docs/command-reference/remove.md

```diff
@@ -8,15 +8,15 @@ Remove DVC-tracked files or directories from the <abbr>workspace</abbr>.
 usage: dvc remove [-h] [-q | -v] [-o | -p] [-f] targets [targets ...]
 
 positional arguments:
-  targets        DVC-files to remove.
+  targets        `.dvc` files to remove.
 ```
 
 ## Description
 
 This command safely removes data files or directories that are tracked by DVC
```

Comment:
Hello again @efiop sorry to pick on you. Will `remove` work only for .dvc files in 1.0? Or also dvc.yaml outputs somehow? I assume only .dvc files.

Reply:
@jorgeorpinel Oh, replied above too. It works on `stages` and `.dvc` files. So yes, it works with `dvc.yaml` too.

Reply:
Moved to https://github.com/iterative/dvc.org/pull/1403#discussion_r437021912